portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsCustomAsset.testcase

```diff
@@ -351,8 +351,13 @@ definition {
 	}
 
 	@description = "Feature ID: LRAC-7532 | Automation ID: LRAC-8543 | Test Summary: Delete one report out of many created"
+	@ignore = "true"
 	@priority = 3
 	test CanDeleteOneReportOutOfManyCreated {
+
+		// AC Refactor: Data doesn't appear on time
+		// AC Refactor ticket: LRAC-13994
+
```

Comment:
Hey @joshitagaki, I see this test is passing constantly in the release routine, why should we disable it? I think we can continue with this case.

[History](https://testray.liferay.com/home/-/testray/case_results/2345385164/history?tab=result&hideFilterPin=true)

If you agree to continue with it in the routine, then we need to update it so that at the end of the test it verifies that the deleted report is not present after accessing the custom asset. You could create a ticket for this modification or make this update using this PR

Reply:
Thanks I was going off of the upstream routine I was analyzing and wasn't going through all the tickets history. I agree looking at this test it looks like it was a fluke behavior that has appeared but very infrequently. We can probably remove this one from being ignored